content/docs/getting-started/create-database.md

```diff
@@ -71,6 +83,18 @@ curl -X PUT -H 'Content-Type: application/json' \
     -d '{"dbaPassword": "changeIt"}'
 ```
 
+{{< /tab >}}
+{{< tab "terraform" >}}
+
+```terraform
+resource "nuodbaas_database" "db" {
+  organization = nuodbaas_project.proj.organization
+  project      = nuodbaas_project.proj.name
+  name         = "demo"
+  dba_password = "changeIt"

```

Comment:
I suggest adding another resource to generate the password.

```terraform
resource "random_password" "password" {
  length  = 10
  special = true
}
```

Reply:
I wanted to have the password consistent with the other examples.

Reply:
Okay.